docs/blocks-storage/migrate-from-chunks-to-blocks.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Migrate Cortex cluster from chunks to blocks"
+linkTitle: "Migrate Cortex cluster from chunks to blocks"
+weight: 5
+slug: migrate-cortex-cluster-from-chunks-to-blocks
+---
+
+This article describes how to migrate existing Cortex cluster from chunks storage to blocks storage,
+and highlight possible issues you may encounter in the process.
+
+_This document replaces the [Cortex proposal](https://cortexmetrics.io/docs/proposals/ingesters-migration/),
+which was written before support for migration was in place._
+
+## Introduction
+
+This article **assumes** that:
+
+- Cortex cluster is managed by Kubernetes
+- Cortex is using chunks storage
+- Ingesters are using WAL
+
+_If your ingesters are not using WAL, the documented procedure will still apply, but the presented migration script will not work properly without changes, as it assumes that ingesters are managed via StatefulSet._
+
+The migration procedure is composed by 3 steps:
+
+1. [Preparation](#step-1-preparation)
+1. [Ingesters migration](#step-2-ingesters-migration)
+1. [Cleanup](#step-3-cleanup)
+
+_In case of any issue during or after the migration, this document also outlines a [Rollback](#rollback) strategy._
+
+## Step 1: Preparation
+
+Before starting the migration of ingesters, we need to prepare other services.
+
+### Querier and Ruler
+
+_Everything discussed for querier applies to ruler as well, since it shares querier configuration – options prefix is `querier` even when used by ruler._
```

Comment:
```suggestion
_Everything discussed for querier applies to ruler as well, since it shares querier configuration – CLI flags prefix is `-querier` even when used by ruler._
```